ryan_savill/principal_feature_analysis/readme.md

```diff
@@ -0,0 +1,902 @@
+      <td>0.328584</td>
+      <td>-0.005177</td>
+      <td>-0.002429</td>
+    </tr>
+    <tr>
+      <th>image_stdev</th>
+      <td>-0.124373</td>
+      <td>0.597732</td>
+      <td>-0.026935</td>
+      <td>0.059831</td>
+      <td>0.275986</td>
+      <td>0.222842</td>
+      <td>0.207255</td>
+      <td>0.258350</td>
+      <td>0.080221</td>
+      <td>-0.016470</td>
+      <td>0.376158</td>
+      <td>-0.432514</td>
+      <td>0.043550</td>
+      <td>0.222652</td>
+      <td>0.025638</td>
+      <td>-0.004268</td>
+    </tr>
+    <tr>
+      <th>avg distance of 2 closest points</th>
+      <td>0.312403</td>
+      <td>0.040115</td>
+      <td>0.025366</td>
+      <td>-0.113510</td>
+      <td>-0.083929</td>
+      <td>-0.065002</td>
+      <td>0.223752</td>
+      <td>0.060992</td>
+      <td>-0.355553</td>
+      <td>0.261325</td>
+      <td>0.057316</td>
+      <td>-0.050857</td>
+      <td>-0.037707</td>
+      <td>0.038347</td>
+      <td>-0.716577</td>
+      <td>0.324435</td>
+    </tr>
+    <tr>
+      <th>avg distance of 3 closest points</th>
+      <td>0.315714</td>
+      <td>0.031638</td>
+      <td>0.028109</td>
+      <td>-0.084834</td>
+      <td>-0.084732</td>
+      <td>-0.064296</td>
+      <td>0.212046</td>
+      <td>0.076844</td>
+      <td>-0.308744</td>
+      <td>0.228016</td>
+      <td>-0.034887</td>
+      <td>-0.121603</td>
+      <td>-0.049914</td>
+      <td>-0.068178</td>
+      <td>0.120108</td>
+      <td>-0.802665</td>
+    </tr>
+    <tr>
+      <th>avg distance of 4 closest points</th>
+      <td>0.316944</td>
+      <td>0.023361</td>
+      <td>0.033518</td>
+      <td>-0.059729</td>
+      <td>-0.078356</td>
+      <td>-0.068524</td>
+      <td>0.204538</td>
+      <td>0.092240</td>
+      <td>-0.253916</td>
+      <td>0.188049</td>
+      <td>-0.088972</td>
+      <td>-0.156060</td>
+      <td>-0.072032</td>
+      <td>-0.097133</td>
+      <td>0.660690</td>
+      <td>0.499889</td>
+    </tr>
+    <tr>
+      <th>touching neighbor count</th>
+      <td>-0.190434</td>
+      <td>-0.078438</td>
+      <td>-0.284396</td>
+      <td>-0.437245</td>
+      <td>-0.339335</td>
+      <td>0.594522</td>
+      <td>-0.243295</td>
+      <td>0.334427</td>
+      <td>-0.201843</td>
+      <td>0.052019</td>
+      <td>-0.024418</td>
+      <td>0.025692</td>
+      <td>0.012726</td>
+      <td>0.013417</td>
+      <td>0.028237</td>
+      <td>0.006079</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+Now we have a massive matrix of our eigenvectors sorted by eigenvalues, but since every feature has equal influence on the eigenvectors we need to take a subset of matrix A called A_q. 'q' represents the number of eigenvectors we take into account and the value of q will influence how much variance we can explain with the subset of the principal components. We can turn this question around and set a threshold of how much variance should at least be explained and then find the corresponding subset A-q of matrix A:
```

Comment:
```suggestion
Now we have a massive matrix of our eigenvectors sorted by eigenvalues, but since every feature has equal influence on the eigenvectors we need to take a subset of matrix `A` called `A_q`. `q` represents the number of eigenvectors we take into account and the value of `q` will influence how much variance we can explain with the subset of the principal components. We can turn this question around and set a threshold of how much variance should at least be explained and then find the corresponding subset `A-q` of matrix `A`:
```